recipes/zulu-openjdk/all/conanfile.py

```diff
@@ -46,8 +46,6 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.includedirs.append(self._jni_folder)
```

Comment:
```suggestion
        self.cpp_info.includedirs.append(self._jni_folder)
        self.cpp_info.libdirs = []
```

By default, it's ["lib"]. If it's not required, you can set an empty list to disable lib dir.

Reply:
yes, thanks! tested it, this works